### DISCUSSION TITLE
Run Time Errors in Waynes Code

### DISCUSSION THREAD

**Discussion by dmrvb** [twinbasic/twinbasic#1453] [General]
tB is now at the stage where most of my code now compiles, though with lots of yellow warnings, so Its time to see if it runs.
I am getting lots of GUI related RunTime Errors and these appear in bits of code that are part of tB rather than code that I have written.

>>>I find this a bit disconcerting and unhelpful, what do others think????
Sometimes I can work out what is wrong, but in other cases its just not clear

In the screenshot I am seeing an unknown name error from a refresh in code that is not written by me and in a module that is not mine.
I think this relates to setting a label caption from a Form_Load event but that should not give an error. 

and whats going on in the variables window???

on the plus side its **_really good_** that this behind the scenes stuff is written in tB

![image](https://user-images.githubusercontent.com/83877219/211661325-1a041e5f-b2c0-4f6b-9ee9-2476db7fd6fc.png)

**Comment by FullValueRider** [comment]
Do yourself a favour.  In the twinbasic installation, ide directory, edit styles-twinbasic-dark.css to change --debugLineBackColor:			to something like rgb(40, 40, 75).  Then you'll be able to more easily read the highlighted error line.  NB.  This has to be done for each install of twinBasic.
![editeddebugline](https://user-images.githubusercontent.com/81257042/211664950-5d206062-3f4e-47b6-8e2f-12825f762874.jpg)

**Comment by dmrvb** [comment]
I mostly like the default colour scheme, there is scope for a few improvements where the contrast is not perfect, and the mucky brown highlight could be better, but on the big scheme of things these are not big issues.
My early software was done with green text on a black background on a "VDU" and I like it that tB has returned to a black background, I never liked the windows concept of a white background to represent paper.
Actually some of my really earlier programming was done on a teletype...I must be getting old.
